ITE/2/README.adoc

```diff
@@ -0,0 +1,532 @@
+root layouts). By using consistent snapshots and defining custom targets
+metadata, backward compatibility can be maintained.
+. The targets metadata MUST also list the targets metadata of all
+in-toto link metadata files associated with all available packages. Note
+that as the number of packages grows, so will the size of this metadata
+file. Bandwidth considerations will be discussed in the next subsection.
+. Since packages can be added on demand, this role SHOULD use (1, 1)
+online keys, and its metadata SHOULD expire in some function of time
+that represents the average release cycle, for example 1 day or 1 week.
+
+See `package-and-in-toto-metadata-signer.json` for an example of a complete set
+of targets metadata for a particular package signed by this role.
+
+.package-and-in-toto-metadata-signer.json
+[source,json]
+----
+{
+  "signatures": {...},
+  "signed": {
+    ...,
+    "targets": {
+      "foobar-1.0.0.tar.gz": {
+        "custom": {
+          "in-toto": [
+            "in-toto-metadata/root.layout",
+            "in-toto-metadata/0a33cbf67b70f315c0b7a83923bcef35308e986140169950e609e3be38585289/developer.87d52666.link",
+            "in-toto-metadata/0a33cbf67b70f315c0b7a83923bcef35308e986140169950e609e3be38585289/packager.20585de1.link"
+          ]
+        },
+        "hashes": {
+          "sha256": "eb4189fc29d97463822ecd6409677e9a4fcb9d66d9bee392e9f9aece0917fc09",
+          "sha512": "059358c3ecfd97af4b392c384a44d2349d7d5e6f1a78736d891a6eb105a088fff64571fe21c998dfbad113bfcb4ab6ff65a35259fd24fa958f8b17151ff4aefb"
+        },
+        "length": 7206
+      },
+      "in-toto-metadata/0a33cbf67b70f315c0b7a83923bcef35308e986140169950e609e3be38585289/developer.87d52666.link": {
+        "hashes": {
+          "sha256": "3e592d47ab815cb9bd8aa064bbaa2a217c1baa9a55a7f8b73d2da6dfd14a247e",
+          "sha512": "848188de9c1ec1d855ed88d62114f0d4f0f13df6fb5aae77716276a1240cefbe7cf03c1b664e43dc93c3438c82e88f3d910ab3d10d55bfc31a759b7997c4b6cb"
+        },
+        "length": 132251
+      },
+      "in-toto-metadata/0a33cbf67b70f315c0b7a83923bcef35308e986140169950e609e3be38585289/packager.20585de1.link": {
+        "hashes": {
+          "sha256": "e5076f59e2096fb64deae6b13384575d3d63c1c4f7a42f48d0a238097a8823eb",
+          "sha512": "7f8c1496abca669c3e8cdbfd315e2383bb4fc3386d06258d961a5fe059b2ea9afa410f9924462933ec3c6570f2c3744f13882f61394687bf9de1156e7c6c2357"
+        },
+        "length": 57495
+      }
+    }
+    ...,
+  }
+}
+----
+
+The targets role signs targets metadata about the in-toto root layout,
+and all of the public keys used to verify the layout itself. Finally, it
+makes a terminating delegation of all packages ("`*`") to the
+“package-and-in-toto-metadata-signer'' role. It MUST use a different set
+of (m, n) offline keys from the root role (to ensure that a compromise of
+these keys do not lead to a compromise of the root role as well), where
+it is RECOMMENDED that n >= 2, and m >= ceiling(n / 2). Finally, its
```

Comment:
Can you give more context about these choices?  The reason why you choose these may be very interesting to those wanting to customize your advice for their context.  It's a little bit in between a spec and a best practices guide as it is, so I'm having a hard time wrapping my head around parts of it...